pkgs/applications/networking/cluster/argo/default.nix

```diff
@@ -54,7 +54,10 @@ buildGoModule rec {
 
   postInstall = ''
     for shell in bash zsh; do
-      $out/bin/argo completion $shell > argo.$shell
+      ${if (stdenv.buildPlatform == stdenv.hostPlatform)
+        then "$out/bin/argo"
+        else "${pkgsBuildBuild.argo}/bin/argo"
```

Comment:
Can't we always use the "cross" code path here?

Reply:
That would cause a cyclic dependency.

Reply:
Oh, right!